### PR TITLE
pull request created 

### PR DESCRIPTION
**Pull request by Sartish**
https://guessing-who.netlify.app/

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -250,94 +262,260 @@ const setSecret = () => {
...
       // 👆 add the value from the input here
       category: category,
     }
   } else if (category === 'eye color') {
-    // Set this up your self
+    currentQuestion = {
+      attribute: 'eyeColor',
+      value: questions.value,
+      category: category,
+      }
```
Comment (Ulve17) [inline code/script.js:290]: Good code!
If you want to you could just write category, insted of categoty:category,

File: code/script.js [hunk 2]
```diff
@@ -250,94 +262,260 @@ const setSecret = () => {
...
+      //filtering the existing array charactersInPlay. Person (a parameter), 
+      //used to see if attribute of any of the object in this array, matches what the user asks for (stores in value)
+      charactersInPlay = charactersInPlay.filter((person) => {return person[attribute] === value})
+      showAlertBox();
+        alertBox.innerHTML=""
+        alertBox.innerHTML += 
+        `<div> 
+          <p>Yes, this person is wearing ${attribute}! Keep all that are wearing ${attribute}</p>
+          </div>
+          <button onclick="hideAlertBox()">OK</button>
```
Comment (Ulve17) [inline code/script.js:342]: Very good!

File: code/script.js [hunk 3]
```diff
@@ -250,94 +262,260 @@ const setSecret = () => {
...
+// If the player wants to guess, invoke the checkMyGuess function.
 const guess = (suspect) => {
-  // store the interaction from the player in a variable.
-  // remember the confirm() ?
-  // If the player wants to guess, invoke the checkMyGuess function.
+  confirmBox.style.display ='flex'
+  confirmButton.addEventListener('click' ,() => checkMyGuess(suspect))
+  cancelButton.addEventListener('click', () => {
+  confirmBox.style.display =''
+})
```
Comment (Ulve17) [inline code/script.js:449]: NIce!

File: code/script.js [hunk 4]
```diff
@@ -250,94 +262,260 @@ const setSecret = () => {
...
+  amountOfGuesses.innerHTML = ''
+  winOrLooseBlock.style.display = ''
+  looser.style.display =''
+  board.style.display =''
+  guessClicks++;
+})
+
 // Invokes the start function when website is loaded
-start()
+start();
```
Comment (Ulve17) [inline code/script.js:514]: Really good!

File: code/style.css [hunk 5]
```diff
@@ -177,6 +181,63 @@ button {
...
+  justify-self: center;
+  padding: 20px;
+  background: var(--secondary);
+  color: white;
+  font-size: 16px;
+  font-weight: bold;
+  border-radius: 5px;
+  margin-bottom:500px;
+  width: 40%;
+}
```
Comment (Ulve17) [inline code/style.css:205]: Really nice styling on the alert :)

**Review by Ulve17 — APPROVED** [review]
Hi Sara! Really good job with the project! 
Hopefully i didn't miss anything (I probably did) and I am looking forward to review more of your projects 👍 
/Ulysses

File: code/script.js [hunk 6]
```diff
@@ -2,6 +2,18 @@
...
 const restartButton = document.getElementById('restart')
+const findOutButton = document.getElementById('filter')
+const winOrLooseBlock = document.querySelector('.win-or-lose-wrapper')
+const winner = document.querySelector('.winner')
+const looser = document.querySelector('.looser')
+const alertBox = document.querySelector('.alert')
+const confirmBox = document.querySelector('.confirm')
+const howManyGuesses = document.getElementById('so-many-guesses')
+const tryAgainBtn = document.getElementById('try-again-btn')
+const playAgain = document.getElementById('playAgain')
```
Comment (KarolCislowski) [inline code/script.js:13]: all your `ids` are in kebab-case except this one. What happened? 🙊

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 


Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

File: code/script.js (hunk 6 above)
Comment (Sartish) [reply code/script.js:13]: not sure, maybe brain freeze 🤯 Thanks for noticing now changed  :)

**Pull request closed without merging**